retry/retry.go

```diff
@@ -0,0 +1,46 @@
+package retry
+
+import (
+	"github.com/eapache/go-resiliency/retrier"
+	"github.com/wego/pkg/errors"
+	"net"
+	"net/http"
+	"strings"
+	"time"
+)
+
+type retryClassifier struct{}
+
+func (r *retryClassifier) Classify(err error) retrier.Action {
+	if err == nil {
+		return retrier.Succeed
+	}
+	switch err := err.(type) {
+	case net.Error:
+		return retrier.Retry
+	case *errors.Error:
+		if _, ok := err.Err.(net.Error); ok {
+			return retrier.Retry
+		}
+		switch errors.Code(err) {
+		// retry only on 500,502/504/429/401/403
```

Comment:
@mysqto Please update this comment too or better remove it since the code is too obvious, we don't need this comment.

Reply:
done